edb/edgeql/ast.py

```diff
@@ -461,7 +461,13 @@ class UpdateQuery(SubjStatement):
 
 
 class DeleteQuery(SubjStatement):
+    # The extra fields are the same as for SELECT, because
+    # "DELETE ..." is a sugar for "DELETE (SELECT ...)"
```

Comment:
Make a common parent class for SelectQuery and DeleteQuery

Reply:
Added a common parent.